Jenkinsfile

```diff
@@ -52,6 +52,7 @@ pipeline {
       }
       steps {
         deploy("adminusers", "test", null, true)
+        deployEcs("adminusers", "test", null, true, true)
```

Comment:
This line is
```
deploy adminusers on ecs
with the test aws account
without any tags
tag it after deployment
explicitly run tests
```

this is correct but both lines will run the tests twice (afaik) hence the change proposed above